### PR TITLE
[FLINK-11528][docs] Translate the "Use Cases" page into Chinese and fix some typos

### PR DESCRIPTION
**Pull request by xccui**
This PR translates the `usecases.md` page. Also, I fix two typos in the original English page.

### CODE REVIEW

File: usecases.zh.md [hunk 1]
```diff
@@ -4,102 +4,101 @@ title: "应用场景"
 
 <hr />
 
-Apache Flink is an excellent choice to develop and run many different types of applications due to its extensive features set. Flink's features include support for stream and batch processing, sophisticated state management, event-time processing semantics, and exactly-once consistency guarantees for state. Moreover, Flink can be deployed on various resource providers such as YARN, Apache Mesos, and Kubernetes but also as stand-alone cluster on bare-metal hardware. Configured for high availability, Flink does not have a single point of failure. Flink has been proven to scale to thousands of cores and terabytes of application state, delivers high throughput and low latency, and powers some of the world's most demanding stream processing applications.
+Apache Flink 功能强大，支持开发和运行多种不同种类的应用程序。它的主要特性包括：批流一体化、精密的状态管理、事件时间支持以及精确一次的状态一致性保障等。Flink 不仅可以运行在包括 YRAN、Apache Mesos、Kubernetes 在内的多种资源管理框架上，还支持在裸机集群上独立部署。在启用高可用选项的情况下，它不存在单点失效问题。事实证明，Flink 已经可以扩展到数千核心，其状态可以达到 TB 级别，且仍能保持高吞吐、低延迟的特性。世界各地有很多要求严苛的流处理应用都运行在 Flink 之上。
```
Comment (wuchong) [inline usecases.zh.md]: minor: YRAN should be YARN

File: usecases.zh.md [hunk 2]
```diff
@@ -4,102 +4,101 @@ title: "应用场景"
...
+Flink 中针对事件驱动应用的明星特性当属 savepoint。Savepoint 是一个一致性的状态映像，它可以用来初始化任意状态兼容的应用。在完成一次 savepoint 后，即可放心对应用升级或扩容，还可以启动多个版本的应用来完成 A/B 测试。
 
-### What are typical event-driven applications?
+### 典型的事件驱动型应用实例
 
 * <a href="https://sf-2017.flink-forward.org/kb_sessions/streaming-models-how-ing-adds-models-at-runtime-to-catch-fraudsters/">Fraud detection</a>
 * <a href="https://sf-2017.flink-forward.org/kb_sessions/building-a-real-time-anomaly-detection-system-with-flink-mux/">Anomaly detection</a>
 * <a href="https://sf-2017.flink-forward.org/kb_sessions/dynamically-configured-stream-processing-using-flink-kafka/">Rule-based alerting</a> 
 * <a href="https://jobs.zalando.com/tech/blog/complex-event-generation-for-business-process-monitoring-using-apache-flink/">Business process monitoring</a>
 * <a href="https://berlin-2017.flink-forward.org/kb_sessions/drivetribes-kappa-architecture-with-apache-flink/">Web application (social network)</a>
```
Comment (wuchong) [inline usecases.zh.md]: Should we also translate these titles ?

File: usecases.zh.md [hunk 3]
```diff
@@ -4,102 +4,101 @@ title: "应用场景"
...
+Flink 为持续流式分析和批量分析都提供了良好的支持。具体而言，它内置了一个符合 ANSI 标准的 SQL 接口，将批、流查询的语义统一起来。无论是在记录事件的静态数据集上还是实时事件流上，相同 SQL 查询都会得到一致的结果。同时 Flink 还支持丰富的用户自定义函数，允许在 SQL 中执行定制化代码。如果还需进一步定制逻辑，可以利用 Flink DataStream API 和 DataSet API 进行更低层次的控制。此外，Flink 的 Gelly 库为基于批量数据集的大规模高性能图分析提供了算法和构建模块支持。
 
-### What are typical data analytics applications?
+### 典型的数据分析应用实例
 
 * <a href="http://2016.flink-forward.org/kb_sessions/a-brief-history-of-time-with-apache-flink-real-time-monitoring-and-analysis-with-flink-kafka-hb/">Quality monitoring of Telco networks</a>
 * <a href="https://techblog.king.com/rbea-scalable-real-time-analytics-king/">Analysis of product updates &amp; experiment evaluation</a> in mobile applications
 * <a href="https://eng.uber.com/athenax/">Ad-hoc analysis of live data</a> in consumer technology
 * Large-scale graph analysis
```
Comment (wuchong) [inline usecases.zh.md]: same as above

File: usecases.zh.md [hunk 4]
```diff
@@ -4,102 +4,101 @@ title: "应用场景"
...
+### Flink 如何支持数据管道应用？
 
-Many common data transformation or enrichment tasks can be addressed by Flink's SQL interface (or Table API) and its support for user-defined functions. Data pipelines with more advanced requirements can be realized by using the DataStream API which is more generic. Flink provides a rich set of connectors to various storage systems such as Kafka, Kinesis, Elasticsearch, and JDBC database systems. It also features continuous sources for file systems that monitor directories and sinks that write files in a time-bucketed fashion.
+很多常见的数据转换和增强操作可以利用 Flink 的 SQL 接口（或 Table API）及用户自定义函数解决。如果数据管道有更高级的需求，可以选择更通用的 DataStream API 来实现。Flink 为多种数据存储系统（如：Kafka、Kinesis、Elasticsearch、JDBC数据库系统等）内置了连接器。同时它还提供了文件系统的连续型数据源及数据汇，可用来监控目录变化和以时间分区的方式写入文件。
 
-### What are typical data pipeline applications?
+### 典型的数据管道应用实例
 
 * <a href="https://data-artisans.com/blog/blink-flink-alibaba-search">Real-time search index building</a> in e-commerce
 * <a href="https://jobs.zalando.com/tech/blog/apache-showdown-flink-vs.-spark/">Continuous ETL</a> in e-commerce
```
Comment (wuchong) [inline usecases.zh.md]: same as above

**Comment by xccui** [comment]
Hi @wuchong, thanks for the review. I've updated the PR according to your comments.

Best, Xingcan

**Comment by wuchong** [comment]
Looks good to me.

+1 to merge.

**Comment by xccui** [comment]
Thanks for the review @wuchong. Merging this...

**Pull request merged**